### DISCUSSION TITLE
Cannot activate Copilot in vscode

### DISCUSSION THREAD

**Discussion by yinweijie** [community/community#19499] [Copilot Conversations] [answered]
![image](https://user-images.githubusercontent.com/23356124/175823193-94c5123f-f924-48c4-8c1e-c4d72f59dc44.png)

[ERROR] [default] [2022-06-26T15:54:44.007Z] GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo EAI_AGAIN api.github.com"

**Comment by sardorb3k** [comment]
disable and reinstall or on your ISP
there is a problem

**Comment by D7EAD** [comment, marked as answer]
Hi!

Are you behind any particular network firewall or VPN that may be affecting your connection? If not, have you tried disabling and re-enabling the extension?

Please refer to [this](https://github.com/github-community/community/discussions/7704) similar discussion if you continue to have problems.

**Comment by yinweijie** [comment]
I've tried to reinstall the plugin, but the problem persists.

SardorBek Sattarov ***@***.***> 于2022年6月27日周一 03:03写道：

**Reply by D7EAD** [reply]
This particular error, EAI_AGAIN of `getaddrinfo` is a DNS lookup timeout error. Is there anything you can think of that may be causing DNS-related issues between yourself and your network?

**Reply by D7EAD** [reply]
In particular, the error seems to be caused when the DNS server resolving your request responds that it cannot currently do so. 

As a result, I'd try changing DNS servers to check if that helps. Furthermore, if this does not help, I'd set `api.github.com`'s IP address in your hosts file, or the equivalent.

**Comment by MAhmedDev** [comment]
On linux : running this solved the issue for me. 

`sudo sh -c "echo nameserver 8.8.8.8 > /etc/resolv.conf"`

Source: https://askubuntu.com/questions/249203/what-does-sudo-echo-nameserver-8-8-8-8-etc-resolv-conf-do
